
{'date': 'Tue Sep 19 2023 11:58:27.168 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  protao = Elemento(PORTAO_BRONZE), x=10, y=10, w=100, h=100, cena=oceano)
                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 19 2023 11:58:41.23 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    IlhaProibida()
  module <module> line 10
    protao = Elemento(PORTAO_BRONZE, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'PORTAO_BRONZE' is not defined
'''},
{'date': 'Tue Sep 19 2023 11:59:02.702 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    IlhaProibida()
  module <module> line 10
    protao = Elemento(PORTAO_BRONZE, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'PORTAO_BRONZE' is not defined
'''},
{'date': 'Tue Sep 19 2023 12:28:42.810 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 23
  def __init__(self
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 19 2023 12:28:57.39 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 27
    Peao()
  module <module> line 24
    pawn = Elemento(PAWN, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Sep 26 2023 12:06:49.239 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(loca, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:07:27.206 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''test
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos += (Terreno(local, position, 10))
TypeError: 'Terreno' object is not iterable
'''},
{'date': 'Tue Sep 26 2023 12:08:09.192 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(local, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:08:23.493 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(local, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:14:03.778 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 36
    IlhaProibida()
  module <module> line 20
    self.terrenos = [Terreno(l, p, i * 100 + 10, oceano) for i, l, p in enumerate(info_terrenos)]
ValueError: need more than 2 values to unpack
'''},
{'date': 'Tue Sep 26 2023 12:28:00.524 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 45
  def mover(x, y)
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 26 2023 12:41:37.879 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Sep 26 2023 12:42:38.476 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Sep 26 2023 12:43:28.879 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Sep 26 2023 12:54:25.488 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 53
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Oct 10 2023 09:47:58.941 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 53
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Oct 10 2023 12:05:10.976 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 53
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Mon Oct 23 2023 21:35:27.273 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  info_terrenos = 
                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Oct 23 2023 21:36:06.265 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
                                                                                                                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Oct 23 2023 21:36:48.931 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
                                                                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Oct 23 2023 21:39:40.694 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 107
    IlhaProibida()
  module <module> line 52
    self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
NameError: free variable 'row' referenced before assignment in enclosing scope
'''},
{'date': 'Mon Oct 23 2023 21:39:49.760 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 107
    IlhaProibida()
  module <module> line 52
    self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
NameError: free variable 'row' referenced before assignment in enclosing scope
'''},
{'date': 'Mon Oct 23 2023 21:39:53.162 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 107
    IlhaProibida()
  module <module> line 52
    self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
NameError: free variable 'row' referenced before assignment in enclosing scope
'''},
{'date': 'Mon Oct 23 2023 21:47:18.772 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''[['https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg', 'https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg'], ['https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg', 'https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg'], ['https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg', 'https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg'], ['https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg', 'https://imgur.com/BL6lB7H.jpg', 'https://imgur.com/tLDbzd2.jpg']]
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 108
    IlhaProibida()
  module <module> line 53
    self.terrenos = [Terreno(cena=oceano, posy=py*110 + 10, posx=px*110+10, local=lc)
NameError: free variable 'row' referenced before assignment in enclosing scope
'''},
{'date': 'Tue Oct 24 2023 10:08:02.512 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    IlhaProibida()
  module <module> line 46
    self.monta_tabuleiro_oceano()
  module <module> line 51
    info_terrenos = [
NameError: name 'NONE' is not defined
'''},
{'date': 'Tue Oct 24 2023 10:14:07.64 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    IlhaProibida()
  module <module> line 48
    self.terrenos[1][1].ocupa(self.peao)
AttributeError: 'Terreno' object has no attribute '__getitem__'
'''},
{'date': 'Tue Oct 24 2023 10:14:59.550 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    IlhaProibida()
  module <module> line 46
    self.monta_tabuleiro_oceano(self)
TypeError: monta_tabuleiro_oceano() takes 1 positional argument but more were given
'''},
{'date': 'Tue Oct 24 2023 10:16:31.759 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    IlhaProibida()
  module <module> line 48
    self.terrenos[1,1].ocupa(self.peao)
TypeError: list indices must be integer, not tuple
'''},
{'date': 'Tue Oct 24 2023 10:21:42.795 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 80
  peao.mover(self.posx + 10, self.posy + 10 self)
                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Oct 30 2023 19:44:42.629 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 139
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [Terreno(ilha=self, cena=oceano, posy=py*110 + 10, posx=(6-len(row))*55 + px*110+10, local=lc)
NameError: name 'oceano' is not defined
'''},
{'date': 'Mon Oct 30 2023 19:56:02.135 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 99
  abs(origem.posx - destino.posx) <= 1 
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Oct 31 2023 10:18:47.493 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 141
    IlhaProibida()
  module <module> line 44
    self.peao.mover(self.terrenos[0])
  module <module> line 136
    self.terreno.desocupa_e_vai_para(terreno_destino)
  module <module> line 57
    self.peao.move(terreno_destino)
  module <module> line 129
    if not self.terreno == None and not self.terreno.peao == Null:
NameError: name 'Null' is not defined
'''},
{'date': 'Tue Oct 31 2023 10:18:52.802 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 141
    IlhaProibida()
  module <module> line 44
    self.peao.mover(self.terrenos[0])
  module <module> line 136
    self.terreno.desocupa_e_vai_para(terreno_destino)
  module <module> line 57
    self.peao.move(terreno_destino)
  module <module> line 129
    if not self.terreno == None and not self.terreno.peao == Null:
NameError: name 'Null' is not defined
'''},
{'date': 'Tue Oct 31 2023 10:54:15.638 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 51
  self.terrenos = [Terreno(cena=self.oceano, posy=px // 6,
                                                                                                                                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 31 2023 11:03:43.478 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 150
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Oct 31 2023 11:04:32.434 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 150
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Oct 31 2023 11:06:28.613 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 152
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Oct 31 2023 11:07:40.266 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 151
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Oct 31 2023 11:09:04.559 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 151
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    self.terrenos = [
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Oct 31 2023 11:23:05.5 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 154
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    info_terrenos[[(PORTAO_BRONZE, "Portão Bronze")]]
TypeError: list indices must be integer, not list
'''},
{'date': 'Tue Oct 31 2023 11:25:44.818 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 153
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 58
    info_terrenos[[(PORTAO_BRONZE, "Portão Bronze")]]
TypeError: list indices must be integer, not list
'''},
{'date': 'Tue Oct 31 2023 11:26:10.578 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 152
    IlhaProibida()
  module <module> line 42
    self.monta_tabuleiro_oceano()
  module <module> line 60
    self.terrenos += Terreno(
TypeError: 'Terreno' object is not iterable
'''},